play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/checkboxes/Checkboxes.scala

```diff
@@ -61,7 +62,7 @@ object Checkboxes {
         (__ \ "fieldset").readNullable[Fieldset] and
         (__ \ "hint").readNullable[Hint] and
         (__ \ "errorMessage").readNullable[ErrorMessage] and
-        (__ \ "formGroup").readWithDefault[FormGroup](defaultObject.formGroup) and
+        (__ \ "formGroup").readWithDefault[FormGroup](defaultObject.formGroup)(jsonReadsForMultipleInputs) and
```

Comment:
@kyle-bowden even though they don't have examples, we [should update radios and date inputs viewmodels](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-frontend+beforeInputs+path%3A%2Fyaml%2F&type=code) too with this- and add the output to the twirl templates to [match govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/date-input/template.njk#L61-L63) and I guess we could add some extra fixtures to test these examples (if I'm not misremembering that capability)

Reply:
checked just in case and these are all the ones with just "beforeInput" in their output https://github.com/search?q=repo%3Aalphagov%2Fgovuk-frontend+%2FbeforeInput%5B%5Es%5D%2F+path%3A%2Fyaml%2F&type=code